ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -212,7 +212,7 @@ export default function PrivacySettings() {
                     </div>
                   ) : (
                     <Button
-                      type="secondary"
+                      variant={BUTTON_VARIANT.SECONDARY}
                       rounded
                       large
```

Comment:
Make sure to migrate the props appropriately these props are not accepted by the `Button` component from the component library.
```suggestion
                      size={BUTTON_SIZES.LG}
```